### PR TITLE
Added Metal 2 Port and Metal 2 Service Profile Tests

### PR DESCRIPTION
**Pull request by jkallem-equinix**

### CODE REVIEW

File: examples/metal-nimf-2-port-connection/outputs.tf
Comment (srushti-patl) [inline examples/metal-nimf-2-port-connection/outputs.tf:9]: nit: Add extra space

File: examples/metal-nimf-2-port-connection/terraform.tfvars.example
Comment (srushti-patl) [inline examples/metal-nimf-2-port-connection/terraform.tfvars.example:25]: nit: Remove extra space

File: examples/metal-nimf-2-port-connection/variables.tf
Comment (srushti-patl) [inline examples/metal-nimf-2-port-connection/variables.tf:89]: nit: Add extra space

File: examples/metal-nimf-2-service-profile-connection/README.md
Comment (srushti-patl) [inline examples/metal-nimf-2-service-profile-connection/README.md:1]: Service profile should be written like this: Generic Service Profile. Replace these words in the entire README.md file.

File: examples/metal-nimf-2-service-profile-connection/main.tf
Comment (srushti-patl) [inline examples/metal-nimf-2-service-profile-connection/main.tf:41]: nit:remove extra space

File: examples/metal-nimf-2-service-profile-connection/outputs.tf
Comment (srushti-patl) [inline examples/metal-nimf-2-service-profile-connection/outputs.tf:9]: nit: Add extra space

File: examples/metal-nimf-2-service-profile-connection/variables.tf
Comment (srushti-patl) [inline examples/metal-nimf-2-service-profile-connection/variables.tf:91]: Add zside_location description

**Review by srushti-patl — CHANGES_REQUESTED** [review]
Requested few changes.

**Review by srushti-patl — APPROVED** [review]
LGTM. Waiting for @thogarty's approval.

File: examples/metal-nimf-2-port-connection/versions.tf
Comment (thogarty) [inline examples/metal-nimf-2-port-connection/versions.tf:6]: Please update this to 1.36.4. There's a breaking API response for any versions before that in UAT1 and above.

File: examples/metal-nimf-2-service-profile-connection/versions.tf
Comment (thogarty) [inline examples/metal-nimf-2-service-profile-connection/versions.tf:6]: Please update this to 1.36.4. There's a breaking API response for any versions before that in UAT1 and above.

**Review by thogarty — CHANGES_REQUESTED** [review]
Requested changes on version numbers to avoid issues in UAT1 and above. Otherwise the change looks good. Thanks, @jkallem-equinix !

File: examples/metal-nimf-2-port-connection/versions.tf
Comment (jkallem-equinix) [reply examples/metal-nimf-2-port-connection/versions.tf]: Done

File: examples/metal-nimf-2-service-profile-connection/versions.tf
Comment (jkallem-equinix) [reply examples/metal-nimf-2-service-profile-connection/versions.tf]: Done

**Review by thogarty — APPROVED** [review]
LGTM. Thank you @jkallem-equinix !

**Pull request merged**
